src/runtime_src/core/edge/drm/zocl/sched_exec.c

```diff
@@ -770,10 +770,13 @@ configure(struct sched_cmd *cmd)
 
 	cfg = (struct ert_configure_cmd *)(cmd->packet);
 
-	if (exec->configured != 0) {
-		DRM_WARN("Reconfiguration not supported\n");
-		return 1;
-	}
+	/*
+	 * In ert mode, no ert config cmd for the same xclbin will be coming
+	 * from host. so always unconfig ert when there is ert config cmd
+	 * arriving
+	 */
+	if (zdev->ert)
+		sched_reset_scheduler(zdev->ddev)
 
```

Comment:
@larry9523  is this change safer for versal too? I think we don't want to support reconfig?

Reply:
xocl is responsible to make sure only one config command is sent per xclbin